src/librustc_codegen_ssa/meth.rs

```diff
@@ -91,22 +91,27 @@ pub fn get_vtable<'tcx, Cx: CodegenMethods<'tcx>>(
         (&[]).iter()
     };
 
-    let methods = methods.cloned().map(|opt_mth| {
-        opt_mth.map_or(nullptr, |(def_id, substs)| {
-            callee::resolve_and_get_fn_for_vtable(cx, def_id, substs)
+    let methods = methods.cloned().map(|trait_methods| {
```

Comment:
Could use a comment wrt. what this does.

Reply:
Done